pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -214,7 +214,7 @@ def _on_evaluation_end(self, *args: Any, **kwargs: Any) -> None:
             self.trainer.call_hook("on_validation_end", *args, **kwargs)
 
         # reset any `torchmetrics.Metric` and the logger connector state
-        self.trainer.logger_connector.reset_results(metrics=True)
```

Comment:
The `metrics=...` flag is now unused, remove it?

Reply:
Done !